packages/manager/apps/dedicated/client/app/index.html

```diff
@@ -75,9 +75,7 @@
                         data-sidebar-namespace="'account'"
                     ></div>
 
-                    <div
-                        class="col-md-9 col-md-offset-3 col-lg-10 col-lg-offset-2 px-0"
-                    >
+                    <div class="col-lg-10 offset-lg-2 px-0">
```

Comment:
`md` not required ?

Reply:
Nope, other CSS rule discard side-menu before `md` rule. :)

Reply:
I'm quite confused with this change as the same component and same configuration is applied on other universe (e.g: web https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/index.html#L68) and it doesn't seem to cause any issue or if it does maybe it should be applied on other universes

Reply:
for apply on other universe, we need to import same version of bootstrap everywhere :/